### PR TITLE
Fix styling and link to code

### PR DESCRIPTION
**Pull request by edwardhorsford**

### CODE REVIEW

File: app/posts/register-trainee-teachers/2022-01-20-preventing-browser-auto-fill.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Ultimately, it seems that if browsers see the same combination of field name / t
 
 ## Our solution
 
-Our current working solution is to randomise the name attribute on page load, and restore it before submitting it. As the name is randomised each time, browsers will treat it as a new field they’ve not seen before. 
+Our current working solution ([view on GitHub(https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/webpacker/scripts/global/disable-browser-autofill.js)]) is to randomise the name attribute on page load, and restore it before submitting it. As the name is randomised each time, browsers will treat it as a new field they’ve not seen before.
```
Comment (ralph-hawkins) [inline app/posts/register-trainee-teachers/2022-01-20-preventing-browser-auto-fill.md]:
```suggestion
Our current working solution ([view on GitHub](https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/webpacker/scripts/global/disable-browser-autofill.js)) is to randomise the name attribute on page load, and restore it before submitting it. As the name is randomised each time, browsers will treat it as a new field they’ve not seen before. 
```

File: app/posts/register-trainee-teachers/2022-01-20-preventing-browser-auto-fill.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ Many users’ browsers default to auto-filling user data — this is often usefu
 
 ## When auto-fill is not suitable
 
-For Register, auto-fill is not relevant and it makes using Register harder. Our users are filling in their trainee’s details, not their own. We do not want the details of _other_ trainees or the Register user showing up when they’re adding a new trainee.
+For Register, auto-fill is not relevant and it makes using Register harder. Our users are filling in their trainee’s details, not their own. We do not want the details of other trainees or the Register user showing up when they’re adding a new trainee.
```
Comment (ralph-hawkins) [inline app/posts/register-trainee-teachers/2022-01-20-preventing-browser-auto-fill.md:11]: yes! that forced italics was horrible

**Review by ralph-hawkins — CHANGES_REQUESTED** [review]

**Comment by ralph-hawkins** [comment]
It'd be nice if you prevented wrapping on "auto-fill", "auto-filled", "non-standard" and "client-side".

Screenshot 2022-01-21 at 10 49 03

File: app/posts/register-trainee-teachers/2022-01-20-preventing-browser-auto-fill.md (hunk 2 above)
Comment (edwardhorsford) [reply app/posts/register-trainee-teachers/2022-01-20-preventing-browser-auto-fill.md:11]: Unintentional - force of habbit!

File: app/posts/register-trainee-teachers/2022-01-20-preventing-browser-auto-fill.md [hunk 3]
```diff
@@ -34,14 +34,15 @@ Ultimately, it seems that if browsers see the same combination of field name / t
 
 ## Our solution
 
-Our current working solution is to randomise the name attribute on page load, and restore it before submitting it. As the name is randomised each time, browsers will treat it as a new field they’ve not seen before. 
+Our current working solution ([view on GitHub(https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/webpacker/scripts/global/disable-browser-autofill.js)]) is to randomise the name attribute on page load, and restore it before submitting it. As the name is randomised each time, browsers will treat it as a new field they’ve not seen before.
```
Comment (edwardhorsford) [inline app/posts/register-trainee-teachers/2022-01-20-preventing-browser-auto-fill.md]:
```suggestion
Our current working solution ([view on GitHub](https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/webpacker/scripts/global/disable-browser-autofill.js)]) is to randomise the name attribute on page load, and restore it before submitting it. As the name is randomised each time, browsers will treat it as a new field they’ve not seen before.
```

**Review by ralph-hawkins — APPROVED** [review]

**Pull request merged**